server/requestorserver/server.go

```diff
@@ -561,7 +562,7 @@ func (s *Server) revoke(w http.ResponseWriter, requestor string, request *irma.R
 	}
 	if err := s.irmaserv.Revoke(request.CredentialType, request.Key, issued); err != nil {
 		if err == irma.ErrUnknownRevocationKey {
-			server.WriteError(w, server.ErrorUnknownRevocationKey, request.Key)
+			server.WriteError(w, server.ErrorUnknownRevocationKey, "")
```

Comment:
The revocation key may contain personal data, so logging those would be dangerous.